### PR TITLE
Adding a reference doc for fuzzy operator

### PR DESCRIPTION
**Pull request by alexeyche**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by kkraune** [comment]
@geirst  did we have a review of config names? Text is fine with me

**Comment by alexeyche** [comment]
I got weird test fail, can someone help with that?

**Comment by vekterli** [comment]
`verify-guides` build error looks like it's due to something entirely unrelated to this PR, so it can probably just be ignored.
```
15:09:05 [ERROR] Tests run: 3, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 2.541 s <<< FAILURE! - in ai.vespa.example.album.MetalSearcherTest
15:09:05 [ERROR] testAddedOrTerm2  Time elapsed: 1.894 s  <<< ERROR!
15:09:05 com.yahoo.container.di.componentgraph.core.ComponentNode$ComponentConstructorException: Error constructing 'com.yahoo.container.logging.FileConnectionLog': null
15:09:05 Caused by: java.lang.NoClassDefFoundError: com/sun/jna/Platform
15:09:05 	at com.yahoo.nativec.NativeC.loadLibrary(NativeC.java:8)
15:09:05 	at com.yahoo.nativec.PosixFAdvise.<clinit>(PosixFAdvise.java:12)
15:09:05 	at com.yahoo.io.NativeIO$InitResult.<init>(NativeIO.java:31)
15:09:05 	at com.yahoo.io.NativeIO.<clinit>(NativeIO.java:21)
15:09:05 	at com.yahoo.container.logging.LogFileHandler$LogThread.<init>(LogFileHandler.java:198)
15:09:05 	at com.yahoo.container.logging.LogFileHandler.<init>(LogFileHandler.java:58)
15:09:05 	at com.yahoo.container.logging.LogFileHandler.<init>(LogFileHandler.java:52)
15:09:05 	at com.yahoo.container.logging.ConnectionLogHandler.<init>(ConnectionLogHandler.java:18)
15:09:05 Caused by: java.lang.ClassNotFoundException: com.sun.jna.Platform
15:09:05 	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
15:09:05 	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
15:09:05 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
15:09:05 	... 8 more
```

File: en/reference/query-language-reference.html [hunk 1]
```diff
@@ -406,6 +406,38 @@ <h2 id="where">where</h2>
...
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+              <td><a href="#maxeditdistance">maxEditDistance</a></td>
+              <td>An inclusive upper bound of edit distance between query and string attribute (default is 2).</td>
+            </tr>
+            <tr>
+              <td><a href="#prefixlength">prefixLength</a></td>
+              <td>Number of characters that are considered frozen, so the fuzzy match will be performed only top of the suffix left. Default is 0 (i.e. <code>fuzzy</code> will match across whole query)</td>
```
Comment (geirst) [inline en/reference/query-language-reference.html:431]: I am ensure about the meaning of this sentence: "... performed only top of the suffix left". Is 'top' the right word here?

**Comment by geirst** [comment]
> @geirst did we have a review of config names? Text is fine with me

No, we haven't decided in an architect meeting yet. I will ensure we do that this week.

File: en/reference/query-language-reference.html (hunk 1 above)
Comment (alexeyche) [reply en/reference/query-language-reference.html]: Yeah, good point, changed the wording

**Review by geirst — APPROVED** [review]
👍

**Comment by geirst** [comment]
The decision in the architect meeting was as proposed in this reference doc.

**Comment by alexeyche** [comment]
> The decision in the architect meeting was as proposed in this reference doc.

Yay!

**Pull request merged**